ROADMAP.md

```diff
@@ -0,0 +1,135 @@
+# ROADMAP
+
+See [README.md](README.md) for developer getting start instructions.
+
+## Minimal complete pipeline
+
+* `heareval/baseline.py` - Simple unlearned embedding algorithm
+following the API.
+
+* `heareval/tasks/` - One time preprocessing of tasks into train +
+test WAVs and `train/test.csv`.
```

Comment:
I might be explicit about that fact that this is to put everything into a nice common format.